### ISSUE TITLE
after deleting and recreating friends

### ISSUE THREAD

**Issue by Saro-Markosyan** [HieuApp/android-firebase-chat#53]
it's creating duplicate friends and give sqlite database error on deleting friend side and java.lang.IndexOutOfBoundsException. 
how remove friends on both side ?

**Comment by haroon47** [comment]
Yes same issue with me, when we try to delete the user, it gets deleted, but when we open the app again, we see the same conversation is still there.any help would be appreciated by the developer.
